cli/src/semgrep/app/scans.py

```diff
@@ -204,6 +212,7 @@ def report_findings(
                 "findings": len(new_matches),
                 "errors": [error.to_dict() for error in errors],
                 "total_time": total_time,
+                "unsupported_exts": ignored_ext_freqs,
```

Comment:
Probably want to convert to some json format since it looks like it prints out a counter object in the tests below